src/sentry/conf/server.py

```diff
@@ -680,7 +680,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("get_suspect_resolutions_releases", routing_key="get_suspect_resolutions_releases"),
     Queue("replays.delete_replay", routing_key="replays.delete_replay"),
     Queue("counters-0", routing_key="counters-0"),
-    Queue("triggers-0", routing_key="triggers-0"),
+    Queue("post-process-errors", routing_key="post-process-errors"),
```

Comment:
Just to be safe, let's duplicate this queue here, and remove `triggers-0` in another pr. I'm not sure if there could be deploy time problems where a worker is unable to process the remainder of the old queue or something along those lines.

Reply:
oh okay, got it 👍 added the `triggers-0` queue back